relayer/pkg/chainlink/keys/keystore.go

```diff
@@ -22,4 +58,117 @@ type NonceManager interface {
 
 	NextSequence(address caigotypes.Hash, chainID string) (*big.Int, error)
 	IncrementNextSequence(address caigotypes.Hash, chainID string, currentNonce *big.Int) error
-}
\ No newline at end of file
+}
+
+const (
+	maxPointByteLen = 32 // stark curve max is 252 bits
+	signatureLen    = 2 * maxPointByteLen
+)
+
+// MemKeystore is an in-memory implementation of the LOOPp Keystore interface
+type MemKeystore struct {
+	mu   sync.Mutex
+	keys map[string]*big.Int
+}
```

Comment:
just curious: is this for testing or something that will be a part of the codebase?

Reply:
yes, this was test code. reorganized now